openwrt-openwisp-monitoring/tests/basic_env.lua

```diff
@@ -0,0 +1,55 @@
+local env={}
+env.io = {
+    popen = function(arg)
+        if arg == 'cat /proc/loadavg' then
+            local f = assert(io.tmpfile())
+            f:write('0.37 0.95 1.23 2/873 56899\n')
+            f:seek('set',0)
+            return f
+        end
+        local f = assert(io.tmpfile())
+        f:write('')
+        f:seek('set',0)
+        return f
+    end,
+    open = function(arg)
+        return nil
+    end
+}
+
+env.ubus = {
+    connect =function()
+        return {
+            call = function(...)
+                local arg = {...}
+                if arg[2]=='system' and arg[3]=='board' then
```

Comment:
white space is off around the comparison operators here, why isn't the qa check spotting this?

Reply:
There is an open issue for this https://github.com/luarocks/luacheck/issues/19

For now, I will check on my IDE